src/notehead.ts

```diff
@@ -107,18 +43,20 @@ export class NoteHead extends Note {
     return Category.NoteHead;
   }
 
-  glyphCode: string;
-
   protected customGlyph: boolean = false;
-  protected stemUpXOffset: number = 0;
-  protected stemDownXOffset: number = 0;
   protected displaced: boolean;
   protected stemDirection: number;
 
   protected line: number;
   protected index?: number;
   protected slashed: boolean;
 
+  // map to correspoding glyph with ledger line
```

Comment:
Typo (correspoding) & formatting. 

Also... I'm not sure what this comment means. :-) :-(

Reply:
second try :)